src/main/java/domain/LottoPrizeCount.java

```diff
@@ -0,0 +1,56 @@
+package domain;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+public class LottoPrizeCount {
+    private static final String FIRST_PRIZE_COUNT = "firstPrizeCount";
+    private static final String SECOND_PRIZE_COUNT = "secondPrizeCount";
+    private static final String THIRD_PRIZE_COUNT = "thirdPrizeCount";
+    private static final String FOURTH_PRIZE_COUNT = "fourthPrizeCount";
+
+    private Map<String, Long> lottoPrizeCount = new HashMap<>();
```

Comment:
LottoPrize 를 사용해보면 어떨까요? 😄 
또한 비슷한 메서드들을 줄이는 방법을 고민해보면 어떨까요?
```suggestion
    private Map<LottoPrize, Long> lottoPrizeCount = new HashMap<>();
```